packages/react-hooks/README.md

```diff
@@ -0,0 +1,37 @@
+# @uifabric/react-hooks
+
+**UI Fabric React hooks**
```

Comment:
This documentation should have a section per hook and for each, a sentence why you as a dev would care about the hook.

Reply:
Lots of hooks to take a look at here: https://github.com/rehooks/awesome-react-hooks

As our understanding and usage evolves and good patterns ermerge, we should consider if breaking these out to other OSS (MIT licensed) libraries makes sense.